docs/managed-tests/plugin-check.md

```diff
@@ -0,0 +1,35 @@
+# Plugin Check Tests
```

Comment:
I'll be the AI skeptic and say I don't love this one; it's both longer and more verbose than the original, while not containing all the same info (which specific checks do we run, and the link to the review guidelines).

Reply:
I've incorporated all the info from the old one, while keeping the "writing style" consistent with the rest of the documentation - which is one of the good things about this rewrite, to consolidate the style of writing. I'm open to any suggestions you might have, and feel free to own this and change it however you see fit, it's completely fine. Let me know if you like this version or not:

![image](https://github.com/user-attachments/assets/b797dd4d-c946-46a3-a50a-0b430c1ffaa5)